services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala

```diff
@@ -0,0 +1,17 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.datastorage.dto.FileContent
+import cromwell.pipeline.womtool.WomTool
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class ProjectFileService(womTool: WomTool)(implicit executionContext: ExecutionContext) {
+
+  def validateFile(fileContent: FileContent): Future[Either[ValidateError, String]] =
+    Future(womTool.validate(fileContent.content)).map {
+      case Left(value) => Left(ValidateError(value.head)) // thinking about change Left on Future.failed
+      case Right(_)    => Right("File is valid")
```

Comment:
What a meaningful explanation :) Why don't we return `Future[Either[ValidateError, Unit]]` instead?

Reply:
Yes, You're right